app/src/main/kotlin/com/exallium/todoapp/MainActivity.kt

```diff
@@ -59,4 +66,12 @@ class MainActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
```

Comment:
I meant that you can change `onOptionsItemSelected(item: MenuItem?)` to `onOptionsItemSelected(item: MenuItem)`